jxl/src/headers/frame_header.rs

```diff
@@ -679,7 +679,7 @@ impl FrameHeader {
             return Err(Error::NonPatchReferenceWithCrop);
         }
         if !self.is444()
-            && ((self.flags & Flags::SKIP_ADAPTIVE_LF_SMOOTHING) != 0)
+            && ((self.flags & Flags::SKIP_ADAPTIVE_LF_SMOOTHING) == 0)
```

Comment:
Oh, is this flag for the missing feature we were discussing?

Reply:
Yes, DC smoothing should be done after the last DC group is decoded.